docs/AADEntitites.md

```diff
@@ -0,0 +1,44 @@
+# Azure Active Directory Entities
+This document describes the configuration of entities create in Azure AD by our [deployment script](../src/deployment/deploy.py)
+
+### OneFuzz Application Registration
+This is the registration of the OneFuzz instance.
+* name : <instance_name>
+* app roles
+    * ManagedNode
+        * value: ManagedNode
+        * Allowed Member types: Applications
+    * CliClient
+        * value: ManagedNode
+        * Allowed Member types: Applications
+* API Permissions
+    * User.Read (Microsoft Graph)
+* scope
+    * user_impersonation
+* Authorized application:
+    * OneFuzz CLI registration
+
+### Onefuzz Application Service Principal
+Service principal linked to the OneFuzz application registration.
+* name: `<instance_name>`
+* Application Id: <OneFuzz Application registration app_id>
```

Comment:
Without escaping or annotation, the angle-bracketed code gets treated as an (invalid) markup tag, and is absent when rendered:
```suggestion
* Application Id: _&lt;OneFuzz Application registration `app_id`&gt;_
```
There are a few similar spots below.

Reply:
```suggestion
* Application Id: `<OneFuzz Application registration app_id>`
```